packages/manager/modules/notifications-sidebar/src/controller.js

```diff
@@ -31,18 +37,46 @@ export default class NotificationsCtrl {
     this.isLoading = true;
     this.numberOfActiveNotifications = 0;
 
+    // Will be bound to the click event on $document
+    this.readAllNotifications = () => {
+      this.toggle = false;
+
+      // Automatically set all unread messages to read
+      // When we close the notifications menu
+      this.NavbarNotifications.readAllNotifications(
+        this.getActiveNotifications(),
+      ).then(() => {
+        this.numberOfActiveNotifications = this.getActiveNotifications().length;
+        this.$rootScope.$emit(
+          'ovh::notifications::count',
```

Comment:
**thought:** Maybe events' name can be sorted into a constant file (not mandatory for now but since they are used at multiple place, it might be interesting).

Reply:
Could be interesting to do it global with the micro front-end architecture :)